Gemfile

```diff
@@ -94,6 +94,8 @@ gem "sidekiq", "~> 4.2"
 
 gem "sidekiq-scheduler", "~> 2.0"
 
+gem 'sinatra', '~> 2.0.2'
```

Comment:
Perhaps leave a comment that this is only needed as a dependency of sendgrid-ruby?